trio/_core/_io_epoll.py

```diff
@@ -230,14 +235,14 @@ def force_wakeup(self) -> None:
     # Return value must be False-y IFF the timeout expired, NOT if any I/O
     # happened or force_wakeup was called. Otherwise it can be anything; gets
     # passed straight through to process_events.
-    def get_events(self, timeout: float) -> list[tuple[int, int]]:
+    def get_events(self, timeout) -> EventResult:
```

Comment:
Why is this and `process_events` missing type annotations? Is it WIP?

Reply:
Yep, this PR isn't about typing those IO modules. I just did a few specifiy ones, because they're imported from `_run`.

Reply:
They were handled in #2743

Reply:
Oh, in the merge you accidentally wiped some types that were added in it though